R/bind_sim_output.R

```diff
@@ -0,0 +1,22 @@
+#' Bind simulated subject-level output together
+#' (from list into data.frames)
+#'
+#' @param res temporary result object (list) in `run_trial()`
+#'
+#' @export
```

Comment:
since this is a user-facing function, i think it would be a good idea to document it a little more thoroughly. e.g., a longer `@description` that makes it clearer how this fits into a work flow, a `@returns` field that indicates what is returned. maybe an `@example` that shows how to use it?

I also notice there aren't any tests for this, we probably do want to add some